resources/views/service/traders/create.blade.php

```diff
@@ -6,8 +6,7 @@
         <div id="main-content">
 
             <h1>Add Trader</h1>
-
-            <p>Add a trader, maybe with users, to a market</p>
+            <p>Add a trader optionally it's users, to a market</p>
```

Comment:
Apologies, the way I read it I think it's the possessive that you want here 🤷‍♂️ 
```suggestion
            <p>Add a trader optionally its users, to a market</p>
```